src/main/webapp/WEB-INF/tags/instructor/studentsList/searchBox.tag

```diff
@@ -0,0 +1,36 @@
+<%@ tag description="instructorStudentList - Student search box" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="data" type="teammates.ui.template.InstructorStudentsListSearchBox" required="true" %>
+<c:set var="TOOLTIP_SEARCH_STUDENT" value="<%= Const.Tooltips.SEARCH_STUDENT %>" />
+<c:set var="SEARCH_KEY" value="<%= Const.ParamsNames.SEARCH_KEY %>" />
+<c:set var="SEARCH_STUDENTS" value="<%= Const.ParamsNames.SEARCH_STUDENTS %>" />
+<c:set var="SEARCH_COMMENTS_FOR_STUDENTS" value="<%= Const.ParamsNames.SEARCH_COMMENTS_FOR_STUDENTS %>" />
+<c:set var="SEARCH_COMMENTS_FOR_RESPONSES" value="<%= Const.ParamsNames.SEARCH_COMMENTS_FOR_RESPONSES %>" />
+<c:set var="USER_ID" value="<%= Const.ParamsNames.USER_ID %>" />
```

Comment:
It should work; scriptlets should work anywhere in a .tag file unless the webapp is set to disable all scriptlets.
(Though of course, we want to eliminate the embedded Java ones. We just don't have a good solution for constants.)